tensorflow_addons/optimizers/lamb.py

```diff
@@ -74,6 +76,15 @@ def __init__(
               decay of learning rate. `lr` is included for backward
               compatibility, recommended to use `learning_rate` instead.
         """
+
+        if "weight_decay_rate" in kwargs:
+            warnings.warn(
+                "weight_decay_rate has been renamed to weight_decay",
```

Comment:
Palese claim an Addons version as in the PR example I've mentioned

Reply:
added